module/applications/advancement/advancement-manager.mjs

```diff
@@ -466,6 +466,7 @@ export default class AdvancementManager extends Application {
 
     // Render the step
     this.step.flow._element = null;
+    this.step.flow.options.manager ??= this;
```

Comment:
Because that is a static method we don't have access to the `AdvancementManager` instance in that location, so adding it would require modifying every place we call it. I figured this was a much more manageable change.

Reply:
Yeah that's fair enough, it's not a big deal either way